apps/librelingo_yaml_loader/tests/test_yaml_loader_load_skills.py

```diff
@@ -0,0 +1,35 @@
+"""
+this file contains tests of the funcion
+librelingo_yaml_loader.yaml_loader._load_skills
+"""
+from pathlib import Path
+import pytest
+
+from librelingo_yaml_loader.yaml_loader import _load_skills
+from librelingo_fakes import fakes
+
+
+@pytest.fixture
+def mock_load_skill(mocker):
+    yield mocker.patch("librelingo_yaml_loader.yaml_loader._load_skill")
+
+
+def test_returns_correct_value(fs, mock_load_skill):
+    # pylint: disable=unused-argument
+    mock_load_skill.return_value = fakes.fake_value()
+    assert _load_skills("foo", ["bar"], fakes.course1) == [mock_load_skill.return_value]
+
+
+def test_handles_every_module(fs, mock_load_skill):
+    # pylint: disable=unused-argument
```

Comment:
Hi @vil02 thanks for the amazing PR!

It looks perfect but I noticed that you had to ignore unused arguments a lot and I realized that this is an issue in all of the pytest files.

I googled a bit and I found that there's a plugin for pylint that solves this and I installed it in this PR: https://github.com/LibreLingo/LibreLingo/pull/2043

can you please update this branch to main, and remove these pylint ignore comments in order to test that it worked? 

:pray: 

thank you! :rocket:

Reply:
Done.